src/tests/helpers/functional_test_case.ts

```diff
@@ -62,6 +62,26 @@ export class FunctionalTestCase extends InternTestCase {
     return offset > -1 && offset < 1
   }
 
+  async observeAttributeChanges(elementId: string, attribute: string) {
```

Comment:
@javan @sstephenson this feels unwieldy, but it was the best idea I could come up with for tracking whether or not a test toggled a boolean attribute. All other ideas were consistently losing the race between submission and subsequent page load that were unsetting the attribute.

Is there a tool in our testing harness that could better provide a seam between submission and response? A `?sleep` query parameter, maybe?

Reply:
Creating a general purpose assertion would be useful in case we wanted to test that submitter buttons are marked as `[disabled]` upon submit and toggled off when the request finished.